RealityMixer/Calibration/Model/Math.swift

```diff
@@ -161,18 +70,46 @@ struct Quaternion {
             z: (num8 - num11) * rhs.x + (num9 + num10) * rhs.y + (1.0 - (num4 + num5)) * rhs.z
         )
     }
+
+    init(rotationMatrix m: SCNMatrix4) {
+        self.init(simd_double4x4(m))
+    }
+
+    init(x: Double, y: Double, z: Double, w: Double) {
+        self.init(vector: .init(x: x, y: y, z: z, w: w))
+    }
 }
 
-struct Matrix3 {
-    let m11: Double
-    let m12: Double
-    let m13: Double
-    let m21: Double
-    let m22: Double
-    let m23: Double
-    let m31: Double
-    let m32: Double
-    let m33: Double
+typealias Matrix3 = simd_double3x3
+
+extension Matrix3 {
+    var m11: Double { self[0, 0] }
+    var m12: Double { self[1, 0] }
+    var m13: Double { self[2, 0] }
+    var m21: Double { self[0, 1] }
+    var m22: Double { self[1, 1] }
+    var m23: Double { self[2, 1] }
+    var m31: Double { self[0, 2] }
+    var m32: Double { self[1, 2] }
+    var m33: Double { self[2, 2] }
+
+    init(
+        m11: Double,
+        m12: Double,
+        m13: Double,
+        m21: Double,
+        m22: Double,
+        m23: Double,
+        m31: Double,
+        m32: Double,
+        m33: Double
+    ) {
+      self.init(rows: [
+        .init(x: m11, y: m12, z: m13),
+        .init(x: m21, y: m22, z: m23),
+        .init(x: m31, y: m32, z: m33)
+      ])
+    }
```

Comment:
I'm following a column-major order, so I'd change this to:

```suggestion
extension Matrix3 {
    var m11: Double { self[0, 0] }
    var m12: Double { self[0, 1] }
    var m13: Double { self[0, 2] }
    var m21: Double { self[1, 0] }
    var m22: Double { self[1, 1] }
    var m23: Double { self[1, 2] }
    var m31: Double { self[2, 0] }
    var m32: Double { self[2, 1] }
    var m33: Double { self[2, 2] }

    init(
        m11: Double,
        m12: Double,
        m13: Double,
        m21: Double,
        m22: Double,
        m23: Double,
        m31: Double,
        m32: Double,
        m33: Double
    ) {
      self.init([
        .init(x: m11, y: m12, z: m13),
        .init(x: m21, y: m22, z: m23),
        .init(x: m31, y: m32, z: m33)
      ])
    }
```